al/index.php

```diff
@@ -121,9 +121,9 @@ function gtag(){dataLayer.push(arguments);}
 <div class="container">
 <div class="row">
 <div class="col-md-12">
-<div class="heading-title-big"><br>It's your coin <br><span>BitcoinZ: The True Bitcoin 2.0</span></div>
-<div class="description-slide">Welcome to the decentralization</div>
-<div class="buttons-section"><a href="<?php echo $t['community-paper-link']; ?>" target="_blank" class="btn dark-btn large-btn"><?php echo $t['Community Paper']; ?></a><a href="https://connect.bitcoinz.global/" class="btn dark-btn large-btn" target="_blank">READ MORE</a><a href="https://bitcoinz.global/donations/" class="btn dark-btn large-btn" target="_blank">Donate</a><br><br><center><script type="text/javascript" src="/api.js"></script><div class="coinmarketcap-currency-widget" data-currency="bitcoinz" data-base="USD" data-secondary="" data-ticker="true" data-rank="true" data-marketcap="true" data-volume="true" data-stats="USD" data-statsticker="false"></div></center>
+<div class="heading-title-big"><br><?= $t['It\'s your coin']; ?><br><span><?= $t['BitcoinZ: The true Bitcoin 2.0']; ?></span></div>
```

Comment:
I would just do because it's easier to read here:
```
$t["It's your coin"];
```

Reply:
Nit picks welcome, thanks!